package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@outreach/client-addon-sdk",
   "license": "MIT",
-  "version": "0.0.124",
+  "version": "0.10",
```

Comment:
I just want to double check that this is intended (seems weird going from 0.0.124 to 0.10)

Reply:
looks like this is indeed intenional, nevermind

Reply:
I was thinking about how sdk with version 0.0.124 feels like "early POC".
0.10 is also bad but 10x less :)

Reply:
SDK v 0.0.x feels to me as "early POC" and 0.x is while still bad 10x better :)